deploy/runtools/runtime_config.py

```diff
@@ -138,26 +155,47 @@ def array_to_lognames(values, prefix):
         command_blkdev_logs = array_to_lognames(all_rootfses, "blkdev-log")
 
         command_bootbinaries = array_to_plusargs(all_bootbinaries, "+prog")
-        zero_out_dram = "+zero-out-dram" if (enable_zerooutdram) else ""
-        disable_asserts = "+disable-asserts" if (disable_asserts_arg) else ""
-        print_cycle_prefix = "+print-no-cycle-prefix" if not enable_print_cycle_prefix else ""
+        zero_out_dram = "+zero-out-dram" if (hostdebug_config.zero_out_dram) else ""
+        disable_asserts = "+disable-asserts" if (hostdebug_config.disable_synth_asserts) else ""
+        print_cycle_prefix = "+print-no-cycle-prefix" if not synthprint_config.cycle_prefix else ""
 
         # TODO supernode support
         dwarf_file_name = "+dwarf-file-name=" + all_bootbinaries[0] + "-dwarf"
 
-        # TODO: supernode support (tracefile, trace-select.. etc)
-        basecommand = f"""screen -S fsim{slotid} -d -m bash -c "script -f -c 'stty intr ^] && sudo ./{driver} +permissive $(sed \':a;N;$!ba;s/\\n/ /g\' {runtimeconf}) +slotid={slotid} +profile-interval={profile_interval} {zero_out_dram} {disable_asserts} {command_macs} {command_rootfses} {command_niclogs} {command_blkdev_logs}  {tracefile} +trace-select={trace_select} +trace-start={trace_start} +trace-end={trace_end} +trace-output-format={trace_output_format} {dwarf_file_name} +autocounter-readrate={autocounter_readrate} {autocounterfile} {command_dromajo} {print_cycle_prefix} +print-start={print_start} +print-end={print_end} {command_linklatencies} {command_netbws}  {command_shmemportnames} +permissive-off {command_bootbinaries} && stty intr ^c' uartlog"; sleep 1"""
-
-        return basecommand
-
+        screen_name = "fsim{}".format(slotid)
+        run_device_placement = "+slotid={}".format(slotid) if self.platform == "f1" else "+device_index={}".format(slotid)
```

Comment:
Why are these plusargs named differently? Why not just pick one?

Reply:
Ah. Good catch. This needs to get removed since we don't support Vitis in this PR.